controllers/volumereplicationgroup_controller.go

```diff
@@ -1838,7 +1855,7 @@ func (v *VRGInstance) validateVRStatus(volRep *volrep.VolumeReplication, state r
 	// if primary, all checks are completed
 	if state == ramendrv1alpha1.Primary {
 		msg = "VolumeReplication resource for the pvc is replicating"
-		v.updateProtectedPVCDataCondition(volRep.Name, VRGConditionReasonReplicating, msg)
+		v.updateProtectedPVCDataCondition(volRep.Name, VRGConditionReasonReady, msg)
```

Comment:
Should the msg change from `replicating` to `ready`?

Reply:
Use the same message as in line [1789](https://github.com/RamenDR/ramen/pull/262/files#diff-428a43c23cf418d4022f864ffab5269355b5e7521d3fb9e5f7bdf6b202b184d8R1789)

Reply:
ACK.

Since the message in line https://github.com/RamenDR/ramen/pull/262/files#diff-428a43c23cf418d4022f864ffab5269355b5e7521d3fb9e5f7bdf6b202b184d8R1789 is for entire VRG and this is for an individual PVC, changing the message to "PVC in the VolumeReplicationGroup is ready for use"